sdk/react/panels/videoView.js

```diff
@@ -304,9 +304,8 @@ var VideoView = React.createClass({
        <ActivityIndicator
           style={loadingStyle}
           size="large"
-      />
-        );
-      }
+       />
+      );
```

Comment:
good catch peter :)

but looking at the previous commit (https://github.com/ooyala/native-skin/commit/fef9a92f6a300796fba70acc2911811dcb121f2f) I can see we removed a layer of conditions, but didn't remove the last brace like akshay said